recipes/counsel-test

```diff
@@ -0,0 +1,3 @@
+(counsel-test
+ :repo "xmagpie/counsel-test"
```

Comment:
I think you will have to be more explicit with this recipe, since the [default recipe](https://github.com/melpa/melpa#recipe-format) will ignore `-test.el` files.

Reply:
Yep, that should have been obvious when building and testing the package as per the checklist. Please let us know if the instructions for that were not clear.

Reply:
I am sorry, that is my bad. All instructions are good, it seems like I have not followed all of them properly, so it is just my lack of attention.